ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -213,6 +213,47 @@ def test_numpy_rfft(
     )
 
 
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("complex"),
```

Comment:
`numpy.rfftn` expects real-value arrays. why are you creating complex arrays as inputs?